src/koreo_tooling/langserver/codelens.py

```diff
@@ -397,18 +407,21 @@ def _code_lens_inputs_action(test_name: str, test_result: TestResults):
     )
 
     indent = (offset + 2) * " "
-    formated_inputs = f"\n{"\n".join(
-        f"{indent}{line}"
-        for line in yaml.dump(spec_inputs).splitlines()
-    )}\n\n"
+    formated_inputs = (
+        f"\n"
+        f"{chr(10).join(f'{indent}{line}' for line in yaml.dump(spec_inputs).splitlines())}"  # noqa: E501
+        f"\n\n"
+    )
```

Comment:
I have no idea if this is the equivalent... but this stuff needs generally done better.